bugbug/tools/code_review_generation.py

```diff
@@ -0,0 +1,754 @@
+import csv
+import json
+import logging
+import re
+
+import anthropic
+import openai
```

Comment:
We shouldn't use these directly, but rely on the generic functions and classes from bugbug/generative_model_tool.py, similarly to code review tool

Reply:
Good point, now relying on `generative_model_tool.py` here: 3234648f00ddc85612c8a8ce94f805814a96e724

Reply:
The qualitative feedback prompt is still there, but will move it to a separate script when moving all evaluation code.